Lib/http/server.py

```diff
@@ -100,6 +100,7 @@
 import socketserver
 import sys
 import time
+import datetime
```

Comment:
Try to support alphabetical order of imports. I.e. add `import datetime` before `import email.utils`.